src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -45,5 +46,12 @@ public class ExportJobConfiguration
         /// Number of pages to be iterated before committing the export progress.
         /// </summary>
         public uint NumberOfPagesPerCommit { get; set; } = 10;
+
+        /// <summary>
+        /// Formats for export jobs.
+        /// </summary>
+#pragma warning disable CA2227 // Collection properties should be read only
+        public IList<ExportJobFormatConfiguration> ExportJobFormats { get; set; }
```

Comment:
>ExportJobFormats [](start = 51, length = 16)

There is no validation for this collection, so there is chance of having two jobs formats with same name, and first one would prevail. It's probably ok, just want to point that out.

Reply:
That is a concern, but since these are user controlled formats if we added any duplicate prevention (like appending something to the name) the user wouldn't know how to request it from the _format parameter. I'll add a note to the documentation in ExportJobFormatConfiguration to not have duplicate names, but I don't think there is a good way to avoid this.